manim/mobject/geometry.py

```diff
@@ -863,6 +864,225 @@ def __init__(self, n=6, **kwargs):
         Polygon.__init__(self, *vertices, **kwargs)
 
 
+class ArcPolygon(VMobject):
+    """A generalized polygon allowing for points to be connected with arcs.
+
+    This version tries to stick close to the way :class:`Polygon` is used. Points can be
+    passed to it directly which are used to generate the according arcs
+    (using :class:`ArcBetweenPoints`). An angle or radius can be passed to it to
+    use across all arcs, but to configure arcs individually an arc_config list has to be
+    passed with the syntax explained below.
+
+    ArcPolygons can be transformed properly into one another as well. Be
+    advised that any arc initialized with 'angle=0' will actually be
+    made like a line, so if a straight secion should seamlessly
+    transform into an arced section, initialize the straight section
```

Comment:
Need a native speaker here, but I'm pretty sure that "arced" is not right here. @eulertour ?

Reply:
We can see if someone else has an issue with that, but it is a legit word used for that purpose: https://www.thefreedictionary.com/arced

Reply:
Woooooah. The more you know, I guess.